data/sql/updates/pending_db_world/rev_1684908860029011348.sql

```diff
@@ -0,0 +1,4 @@
+--
+-- Antu'Sul (Zul'Farrak)
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
```

Comment:
put always a `DELETE FROM` before `INSERT INTO`
https://www.azerothcore.org/wiki/sql-standards#insert-amp-delete

Reply:
Sorry I didn't review that closer - didn't realize I needed to replace the entire set of SAI entries. Fixed now!